src/entity/login.entity.ts

```diff
@@ -9,6 +10,12 @@ export class Login {
     @Column()
     password: string;
 
+    @Column({default: '101'})
+    refreshtoken: string;
```

Comment:
I don't think this should be stored in the database, storing the refresh token is unnecessary. The refresh token should operate exactly like the jwt token just with a longer expire time. The jwt works without database storage. From a security standpoint, storing the refresh token poses potential risks as well as in the event of a breach, those stored tokens can then be used to gain access into our application.